### PR TITLE
CBBR FY2025 tweaks - add manual mapping, clean up file structure

### PR DESCRIPTION
**Pull request by fvankrieken**
Updated source with additional columns. Follow up to #805 

A couple commits are just cleaning up sql, some are deleting a ton of unused logic. Probably easier to go commit by commit, but all relatively simple and (I hope) self-explanatory

### CODE REVIEW

**Comment by fvankrieken** [comment]
Actually this can stay on hold until I revamp how we're taking in manual mappings from PS

File: products/cbbr/03_spatial.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ echo "Assign geometries from facilities data ..."
 run_sql_file sql/spatial_facilities.sql
 
 echo "Assign geometries from manual shapefiles ..."
-run_sql sql/spatial_manual.sql
```
Comment (damonmcc) [inline products/cbbr/03_spatial.sh]: (just curious) did `run_sql` fail quietly or (hopefully) throw an error and stop the build?

**Review by damonmcc — COMMENTED** [review]
latest build seems to be [yesterday](https://github.com/NYCPlanning/data-engineering/actions/runs/9875939965). have there only been rebases since?

File: products/cbbr/03_spatial.sh (hunk 1 above)
Comment (fvankrieken) [reply products/cbbr/03_spatial.sh]: It errored

**Review by damonmcc — APPROVED** [review]

**Pull request merged**